gazebo_plugins/src/gazebo_ros_imu.cpp

```diff
@@ -99,6 +99,15 @@ void GazeboRosIMU::LoadThread()
   else
     this->link_name_ = this->sdf->Get<std::string>("bodyName");
 
+  if (!this->sdf->HasElement("frameId"))
+  {
+    ROS_INFO("imu plugin missing <frameId>, defaults to 'imu'");
+    this->frame_id_ = "imu";
+    return;
+  }
+  else
+    this->frame_id_ = this->sdf->Get<std::string>("frameId");
```

Comment:
So I am a HUGE stickler for *always* using curly braces. The only reason they are not here is to keep the style in line with the upstream code (see the other parameters being loaded), in the event that we ever push to it.